gulpfile.js

```diff
@@ -60,16 +51,41 @@ gulp.task('default', ['clean'], function() {
       path.basename = file_name
       path.extname = ".html"
     }))
+    .pipe(html_replace({
```

Comment:
https://www.npmjs.com/package/gulp-html-replace
https://www.npmjs.com/package/gulp-replace
html-replace will search for <!-- build:<taskname> --> <!-- endbuild --> and replace with what ever you want .
While replace is replace a string(or regex) with a string.
I just feel like html-replace is easier to use because we are explicitly replacing the <head><title></title></head> in the html file.
After second thought, I feel it would be better if we have a defined place holder in <head><title></title></head>(which we currently have).
Like <title>{{HTML-TITLE}} - MDSlides</title>, and use replace to replace "{{HTML-TITLE}} - MDSlides" with file-name.

Date: Tue, 21 Jul 2015 04:23:13 -0700
From: notifications@github.com
To: mdslides-local@noreply.github.com
CC: zhengdavy@hotmail.com
Subject: Re: [mdslides-local] Gulp (#31)

In gulpfile.js:

> @@ -60,16 +51,41 @@ gulp.task('default', ['clean'], function() {
>        path.basename = file_name
>        path.extname = ".html"
>      }))
> +    .pipe(html_replace({

Does html_replace provides additional functionality that replace (see line 48) doesn't provide? Otherwise can we just use replace?


—
Reply to this email directly or view it on GitHub.